docker-storage-setup.1

```diff
@@ -47,14 +44,13 @@ DEVS: A quoted, space-separated list of devices to be used.
       is not specified, then use of the root disk's extra space
       is implied.
 
-VG:   The volume group to use for docker storage.  Defaults to the
+VG:   The volume group to use for container storage.  Defaults to the
       volume group where the root filesystem resides.  If VG is
       specified and the volume group does not exist, it will be
       created (which requires that "DEVS" be nonempty, since we don't
       currently support putting a second partition on the root disk).
 
-      lvm2 version should be same or higher than lvm2-2.02.112 for lvm
-      thin pool functionality to work properly.
+NOTE: lvm2 needs to be lvm2-2.02.112 or later for lvm thin pool functionality to work properly.
```

Comment:
I told @shishir-a412ed to use `Note` instead of `NOTE`. If `NOTE` is better, then we will to change that to make man page uniform.